docs/blog/2024-Q1-community-board.rst

```diff
@@ -0,0 +1,53 @@
+.. post:: Feb 1, 2024
+   :tags: community board, news, report
+   :author: Mikey Ariel
+
+Write the Docs Community Board Quarterly Update Q1-2024
+=======================================================
+
+Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share with you the first quarterly update for 2024, the beginning of our second year of operation. 
+
+To catch up on previous quarterly updates, check out the `community board blog posts <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ archive. 
+
+Q1 quarterly meeting highlights
+-------------------------------
+
+Our Q1 meeting included an informal retrospective on 2023, which was our first year of operation. As a newly grouped collection of team leads, we dedicated some of our efforts to exploring ways to collaborate with each other more proactively and evolving our meta-work tracking and priorities.
+
+Some of the activities that our board members focused on in 2023:
+
+* Reviewed and organized our Kanban board that is used for tracking community-wide task
```

Comment:
I think activities that our members focussed on should probably all be "reviewing and organzing", "launching", etc.

Reply:
I was referring to the community board activities rather than individual member activities, clarified in the intro sentence

Reply:
I get that. But the tense still reads wrong to me. Never mind.